modules/ui/background.js

```diff
@@ -318,6 +316,45 @@ export function uiBackground(context) {
         }
     }
 
+    function getBackgrounds(filter) {
+        return context.background()
+            .sources(context.map().extent(), context.map().zoom(), true)
+            .filter(filter);
+    }
+
+    function chooseBackgroundAtOffset(offset) {
+        var backgrounds = getBackgrounds(function(d) { return !d.isHidden() && !d.overlay; });
+        backgrounds.sort(sortSources);
+        var currentBackground = context.background().baseLayerSource();
+        var foundIndex = -1;
+        for (let index = 0; index < backgrounds.length; index++) {
+            if (backgrounds[index] === currentBackground) {
+                foundIndex = index;
+                break;
+            }
+        }
+        if (foundIndex === -1) {
+            // Can't find the current background, so just do nothing
+            return;
+        }
+
+        var nextBackgroundIndex = (foundIndex + offset + backgrounds.length) % backgrounds.length;
+        var nextBackground = backgrounds[nextBackgroundIndex];
+        if (nextBackground.id === 'custom' && !nextBackground.template()) {
+            nextBackgroundIndex = (nextBackgroundIndex + offset + backgrounds.length) % backgrounds.length;
+            nextBackground = backgrounds[nextBackgroundIndex];
+        }
+        chooseBackground(nextBackground);
+    }
```

Comment:
This function uses `var` and `let`. There are plenty of places where 'var' is still in the iD code, but any new code should be as ES6-ified as possible. Replace `var` with `const` or `let` as appropriate!

Reply:
![image](https://user-images.githubusercontent.com/1225959/86977859-62e63100-c132-11ea-9183-071b3ce783c6.png)

Reply:
yasssssss